docs/src/faqs/migrate_from_fnapi.rst

```diff
@@ -1,180 +1,52 @@
-How Do I Migrate from FnAPI to ``setup.py``?
+.. _migrate-from-fnapi:
```

Comment:
I just realized - this reference should probably be `_migrate-from-setup`. As well, the actual file name should be changed, particularly since that changes the URL.

Actually, I am now thinking it would be really good to keep the original `migrate_from_fnapi.rst` since folks still might want to go from fnapi to setup.py. And then have a new document, like `migrate_from_setup.rst` (or whatever you wish).